README.md

```diff
@@ -66,7 +66,8 @@ You should get a meaningful error message if the authentication fails.
 
 Instead of using echo statements, you can enable/disable debugging
 with the `$wiki->debugMode($boolean)` method.
-Currently only output from the logon process is printed for debugging.
+Debug logging is printed to [standard output](https://en.wikipedia.org/wiki/Standard_output)
+and includes API requests/responses as well as retry messages for lag errors.
```

Comment:
I'm thinking that it may make sense to remove these two paragraphs from the README, in the interests of keeping the content simple (à la hello world) and reducing duplication with `USAGE.md`. Or do you think this is important information to convey for first-time users?

Reply:
I'll assume that you deliberately chose to let the "This example uses echo statements..." part stay. I can see why that might make sense, but I'm just leaving a note for the record, since my original proposal above was to remove it as well.

Reply:
That was somewhat slightly deliberate, yeah. :smile_cat: